openedx/features/teams/tests/test_serializers.py

```diff
@@ -68,10 +68,15 @@ def setUp(self):
 class CreateTeamsSerializerTestCase(SerializersTestBaseClass):
     """ Tests for the create team serializer."""
 
+    def _get_team_dict_data(self):
+        data = self.team.__dict__
+        data['course_id'] = data['course_id'].to_deprecated_string()
+        return data
```

Comment:
Another way of doing it is to use `setUp` function and use `self.data` directly like `...(data=self.data`). Though your solution is fine too.

    def setUp(self):
        super(SerializersTestBaseClass, self).setUp()
		self.data = self.team.__dict__
		data['course_id'] = data['course_id'].to_deprecated_string()

Reply:
I tried the above suggestion but getting following error. 
```
Error
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/openedx/features/teams/tests/test_serializers.py", line 73, in setUp
    data = self.team.__dict__
AttributeError: 'CreateTeamsSerializerTestCase' object has no attribute 'team'
```

Reply:
Secondly we can't do this, as most of the test cases updates or add some attributes in `self.team`. So that data would be missed if we do it in setup method of test class